articles/abstraction-concepts-part2/article.md

```diff
@@ -10,7 +10,7 @@ topics: [languages]
 excerpt_separator: <!--more-->
 images:
 
-  - url: /assets/images/education/python-image-2.jpg
+  - url: /engineering-education/abstraction-concepts-part2/hero.jpg
```

Comment:
@auxesis are we missing the /articles/ path in all of the feature image urls? Shouldn't it be:

/engineering-education/articles/abstraction-concepts-part2/hero.jpg

Reply:
@ninjaginja when we configure the `engineering-education` section in the website repo, we'll strip out the `articles/` part, to preserve current URLs, and retain SEO.

Reply:
Fixed in section-io/section-io.github.io@2d63740d05c389baf9d3dcd22bf961fbbfb092fc.